application/common/models/User.php

```diff
@@ -278,4 +309,20 @@ public function validateAuthKey($authKey)
     {
         return false;
     }
+
+    /**
+     * Get this user as an AuthUser object
+     * @return AuthUser
```

Comment:
Should the @return parameter use the actual class name/path (\Sil\IdpPw\Common\Auth\User) rather than the "use" alias we're using in this file?